jdi-light-html-tests/src/main/java/io/github/com/entities/Contacts.java

```diff
@@ -1,10 +1,17 @@
 package io.github.com.entities;
 
 
+import com.epam.jdi.light.common.Order;
 import com.epam.jdi.tools.DataClass;
 
 public class Contacts extends DataClass<Contacts> {
-    public String name, lastName, acceptConditions, position, passportNumber,
-            passportSeria, description, gender, religion, weather;
+    public String name, lastName, acceptConditions, position, passportNumber;
+    public String passportSeria;
+    @Order(1)
+    public String description;
+    @Order(2)
+    public String gender;
+    public String religion, weather;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5485505)